src/GitcoinGovernor.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+// forgefmt: disable-start
+import { GovernorCountingSimple, Governor } from "openzeppelin-contracts/governance/extensions/GovernorCountingSimple.sol";
+import { GovernorVotesComp, ERC20VotesComp } from "openzeppelin-contracts/governance/extensions/GovernorVotesComp.sol";
+import { GovernorTimelockCompound, ICompoundTimelock } from "openzeppelin-contracts/governance/extensions/GovernorTimelockCompound.sol";
+import { GovernorSettings } from "openzeppelin-contracts/governance/extensions/GovernorSettings.sol";
+// forgefmt: disable-end
+
+/**
+ * @notice The upgraded Gitcoin Governor—Bravo compatible and built from OpenZeppelin.
+ */
+contract GitcoinGovernor is
+  GovernorCountingSimple,
+  GovernorVotesComp,
+  GovernorTimelockCompound,
+  GovernorSettings
+{
+  /// @notice The address of the existing GTC token on Ethereum mainnet which will interop with
+  /// this Governor after the upgrade is complete.
```

Comment:
Do we want to mention things like "after the upgrade is complete" in the code comments? My thinking is no, because the contract itself doesn't care that it's an upgrade, and that won't matter once the upgrade is done, i.e. the upgrade is only temporarily relevant whereas the fact that GTC is the voting token is permanently relevant

Reply:
worded more clearly in 78a11b444add9c6e0fb59e6751995cb55d6c82dd